.circleci/config.yml

```
@@ -39,27 +39,27 @@ test_settings: &test_settings
 version: 2
 jobs:
 
-  py38:
+  py310:
     <<: *test_settings
     docker:
-      - image: python:3.8-buster
+      - image: python:3.10-bookworm
```

I know I just suggested you use 3.10-bookworm, but I realized that this is exclusively for CI, so I think it's suggested to use the CircleCI provided convenience images, in this case: https://circleci.com/developer/images/image/cimg/python.
```suggestion
      - image: cimg/python:3.10
```